README.md

```diff
@@ -23,13 +23,13 @@ A SCSS design system based on Bootstrap, to provide styling and theming to all A
     - `@import "~@ac-ui/design-system/src/page";`
 
 # How do variables get determined?
-- All variables should be declared in the `src/base/_theme.scss` file
+- All variables should be declared in the `src/_theme.scss` file
 - When imported in to a Product, Product variables cascade into Theme Variables
 - Theme variables cascade into Bootstrap variables
 
 # Overview
 - `src/index` -> Entry file into all component, helper, and utility styles
 - `src/page` -> Baseline page style, contains css reset, and utility classes
-- `src/base/theme` -> Baseline theme variables. Product implementation can override.
+- `src/theme` -> Baseline theme variables. Product implementation can override.
```

Comment:
It would be nice to add information about `fonts.scss` over here

And some documentation on how to use it inside projects inside the `fonts.scss` file